### ISSUE TITLE
Support any relative namespace

### ISSUE THREAD

**Issue by goatherd** [packfire-archived/pdc#18]
PDC does resolve relative path with the file namespace only.

It is possible however to substitute any other `use`d namespace.

Example code

``` php
<?php
/* Vendor/Package/x.php */
namespace Vendor\Package;

use Vendor\Exception;

class x {
    public function y()
    {
        throw new Exception\InvalidArgumentException('msg');
    }
}

```

assume that `Vendor/Exception/InvalidArgumentException.php` exists and is well-formed.

PDC will return `Not found: Vendor\Package\Exception\InvalidArgumentException`.

Using Exceptions that way is default for Zend Framework 2 and happens often for common namespaces within Symfony or Doctrine projects.

**Comment by mauris** [comment]
Thanks @goatherd for reporting this. I will look into this later today. Seems to be an interesting use case I didn't really consider much.

**Comment by goatherd** [comment]
Should be fixed with #25.

**Comment by mauris** [comment]
As mentioned, should be fixed with #25
